cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,114 @@
 describe('Web Tables page', () => {
-  it('', () => {
+    const userData = {
+      firstName: 'Anna',
+      lastName: 'Lymorenko',
+      email: 'anyalymorenko@gmail.com',
+      age: '23',
+      salary: '1000',
+      department: 'QA'
+    };
+    beforeEach(() => {
+      cy.visit('https://demoqa.com/webtables');
+    });
 
+    it('should have pagination', () => {
+      cy.get('.-pagination')
+        .should('contain', 'Previous')
+        .and('contain', 'Next');
+      cy.contains('.-pageInfo', 'Page')
+        .should('exist');
+    });
+
+    it('should have row count selection', () => {
+      cy.get('select')
+        .select('10 rows');
+    });
+
+    it('should add a new worker', () => {
+      cy.get('#addNewRecordButton')
+        .click();
+      cy.get('#firstName')
+        .type(userData.firstName);
+      cy.get('#lastName')
+        .type(userData.lastName);
+      cy.get('#userEmail')
+        .type(userData.email);
+      cy.get('#age')
+        .type(userData.age);
+      cy.get('#salary')
+        .type(userData.salary);
+      cy.get('#department')
+        .type(userData.department);
+      cy.get('#submit')
+        .click();
+      cy.get('.rt-tbody')
+        .should('contain', userData.firstName)
+        .and('contain', userData.lastName)
+        .and('contain', userData.email);
+      cy.get('.rt-tbody')
+        .should('contain', userData.age);
+      cy.get('.rt-tbody')
+        .should('contain', userData.salary);
+      cy.get('.rt-tbody')
+        .should('contain', userData.department);
+    });
+  
+    it('should delete worker', () => {
+      cy.get('#delete-record-2')
+        .click();
+    });
+
+    it('should delete all workers', () => {
+      cy.get('#delete-record-3')
+        .click();
+      cy.get('#delete-record-2')
+        .click();
+      cy.get('#delete-record-1')
+        .click();
+    });
```

Comment:
Test case for deleting all workers should include assertions to verify that all workers are no longer present in the table.